evm_trace/gas.py

```diff
@@ -0,0 +1,54 @@
+from typing import Dict, List
+
+from evm_trace.base import CallTreeNode
+
+Address = str
+Calldata = bytes
+GasUsage = int
+
+GasReport = Dict[Address, Dict[Calldata, List[GasUsage]]]
```

Comment:
If we make this a model, we can add validation and such. Else, it may let us do this:

```python
a: Address = "Hiiiii"
g: GasReport = {a: ...}  # Imagine valid Dict[Calldata, List[GasUsage]] for ...
```